syft/messaging/message.py

```diff
@@ -676,3 +676,57 @@ def detail(worker: AbstractWorker, msg_tuple: tuple) -> "WorkerCommandMessage":
             sy.serde.msgpack.serde._detail(worker, command_name),
             sy.serde.msgpack.serde._detail(worker, message),
         )
+
+
+class CryptenInit(Message):
```

Comment:
This seems like the kind of thing PySyft has left to be added on top by other libraries (like Grid.) I'm not opposed to having this functionality in PySyft, but the inconsistency strikes me as pretty weird and a sign that there may be a design issue here.

Reply:
Sorry I didn't get the meaning of "This seems like the kind of thing PySyft has left to be added on top by other libraries"
What would you think would be the best way to proceed here?